packages/@ama-sdk/core/cli/files-pack.cts

```diff
@@ -78,7 +78,7 @@ void (async () => {
             return updateExports();
           }
         })) :
-      globby.sync(path.posix.join(posixBaseDir, glob))
+      globby.sync(glob)
```

Comment:
With this change you will not have issue on Windows because of globby does not support `\`? (or `files` contains always only `/` paths?)

Reply:
glob is not the file just the pattern because mac did not seem to work correctly with the absolute path
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/c5490fc9-41f7-4130-a0b7-585c6fc054aa)